components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityCoreConstants.java

```diff
@@ -60,6 +60,9 @@
 
     public static final String USER_ACCOUNT_STATE = "UserAccountState";
 
+    public static final String AND = "and";
```

Comment:
Can we put these under a sub static class?
We can include all constants related to filtering/searching there.